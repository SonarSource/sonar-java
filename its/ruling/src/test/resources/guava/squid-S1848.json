{
'com.google.guava:guava:src/com/google/common/cache/LongAddables.java':[
37,
],
}
