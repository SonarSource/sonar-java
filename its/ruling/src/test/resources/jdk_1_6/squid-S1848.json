{
'jdk_1_6:java/lang/ref/Finalizer.java':[
72,
],
}
