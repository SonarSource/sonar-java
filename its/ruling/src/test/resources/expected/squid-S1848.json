{
'project:com/sun/corba/se/impl/dynamicany/DynFixedImpl.java':[
186,
190,
],
'project:com/sun/org/apache/bcel/internal/util/Class2HTML.java':[
129,
176,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/SmartTransformerFactoryImpl.java':[
448,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TransformerFactoryImpl.java':[
982,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityManager.java':[
1969,
],
'project:com/sun/org/apache/xerces/internal/impl/dv/xs/AnyURIDV.java':[
63,
],
'project:com/sun/org/apache/xerces/internal/jaxp/DocumentBuilderFactoryImpl.java':[
113,
210,
],
'project:java/lang/ref/Finalizer.java':[
90,
],
'project:javax/swing/plaf/metal/MetalBorders.java':[
971,
],
'project:javax/swing/plaf/metal/MetalLookAndFeel.java':[
1559,
],
'project:javax/swing/plaf/synth/SynthLookAndFeel.java':[
685,
],
}
